impl/client/src/main/javascript/web/pentaho/_core/config/Service.js

```diff
@@ -272,7 +273,34 @@ define([
       },
 
       /** @inheritDoc */
-      selectAsync: function(moduleId, annotationId) {
+      selectAsync: function(moduleId, annotationId, keyArgs) {
+
+        var localConfigPromise = this._selectLocalAsync(moduleId, annotationId);
+        if (!argUtil.optional(keyArgs, "inherit")) {
+          return localConfigPromise;
+        }
```

Comment:
Immediately goes to old behavior, when inheritance is not requested, the default.